### PR TITLE
fix: dataset api variables, postgres indentation issue

### PR DESCRIPTION
**Pull request by ManojKrishnaChintauri**

### CODE REVIEW

File: terraform/modules/helm/dataset_api/variables.tf [hunk 1]
```diff
@@ -1,105 +1,105 @@
...
-}
-
-variable "redis_release_name" {
-  type        = string
-  description = "Release name for Redis installation."
-  default     = "obsrv-redis"
-}
\ No newline at end of file
+# variable "env" {
+#   type        = string
```
Comment (anandp504) [inline terraform/modules/helm/dataset_api/variables.tf:2]: @ManojCKrishna Why are all the variables commented out?

File: terraform/modules/helm/postgresql/postgresql.yaml.tfpl [hunk 2]
```diff
@@ -116,10 +116,10 @@ primary:
           published_date TIMESTAMP NOT NULL,
           UNIQUE (dataset_id)
         );
-      CREATE INDEX IF NOT EXISTS  dataset_source_config_dataset ON dataset_source_config(dataset_id);
+        CREATE INDEX IF NOT EXISTS  dataset_source_config_dataset ON dataset_source_config(dataset_id);
```
Comment (anandp504) [inline terraform/modules/helm/postgresql/postgresql.yaml.tfpl:119]: @ManojCKrishna Why is the indentation required for the index statement?

**Review by anandp504 — CHANGES_REQUESTED** [review]

File: terraform/modules/helm/postgresql/postgresql.yaml.tfpl (hunk 2 above)
Comment (ManojKrishnaChintauri) [reply terraform/modules/helm/postgresql/postgresql.yaml.tfpl:119]: This line should be under obsrv database logic here - https://github.com/Sanketika-Obsrv/obsrv-automation/pull/40/files#diff-b58e82e9ecb675e0dd8424281c5497ce10e18be5ba74cb91ec7a78c4459efff8L39 
So it needs indentation.

**Review by anandp504 — APPROVED** [review]
Approved 👍

**Pull request merged**